<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormIssue</class>
 <widget class="QWidget" name="FormIssue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Issue</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Issue title</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditTitle"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="plainTextEditDescription">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelIssueType">
     <property name="text">
      <string>Issue type</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBoxIssueType"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBoxStatus"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelCreator">
     <property name="text">
      <string>Creator</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEditCreator"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelCreated">
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="labelCreatedValue">
     <property name="text">
      <string>0 seconds</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="labelRelease">
     <property name="text">
      <string>Release</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QComboBox" name="comboBoxRelease"/>
   </item>
   <item row="15" column="1">
    <widget class="QWidget" name="widgetReferences" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidgetReferences">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string comment="Add new reference"/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../graphics/references/add.png</normaloff>../graphics/references/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="Edit reference"/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../graphics/references/edit.png</normaloff>../graphics/references/edit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRemove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="Remove reference"/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../graphics/references/remove.png</normaloff>../graphics/references/remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="labelIdentifier">
     <property name="text">
      <string>Identifier</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QLabel" name="labelIdentifierValue">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="labelReferences">
     <property name="text">
      <string>References</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>plainTextEditDescription</tabstop>
  <tabstop>comboBoxIssueType</tabstop>
  <tabstop>comboBoxStatus</tabstop>
  <tabstop>lineEditCreator</tabstop>
  <tabstop>comboBoxRelease</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
